src/test/java/org/openstreetmap/atlas/checks/validation/intersections/LevelCrossingOnRailwayCheckTest.java

```diff
@@ -0,0 +1,94 @@
+package org.openstreetmap.atlas.checks.validation.intersections;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Unit tests for {@link LevelCrossingOnRailwayCheck}.
+ *
+ * @author atiannicelli
+ */
+public class LevelCrossingOnRailwayCheckTest
```

Comment:
Please add a some unit tests that use the ConfigurationResolver.inLineConfiguration() to check that the 2 configurables work as expected. And please test that the expected fix suggestions are created.

Reply:
Done. I added two new tests to specifically test the two config parameters. Also added verifications for fix suggestions.